boards/microbit_v2/src/main.rs

```diff
@@ -0,0 +1,491 @@
+//! Tock kernel for the Micro:bit v2.
+//!
+//! It is based on nRF52833 SoC (Cortex M4 core with a BLE).
+
+#![no_std]
+// Disable this attribute when documenting, as a workaround for
+// https://github.com/rust-lang/rust/issues/62184.
+#![cfg_attr(not(doc), no_main)]
+#![feature(const_in_array_repeat_expressions)]
```

Comment:
this should no longer be needed (see https://github.com/tock/tock/pull/2215)

Reply:
Done